.gitlab-ci.yml

```diff
@@ -52,10 +52,6 @@ build-and-test:
     - npm --version
     - wasm-prune -V
     - sccache -s
-    # `cargo install` returns an error if there is nothing to update, so have to supress it here temporarily
-    - cargo contract -V
-    - cargo install --git https://github.com/paritytech/cargo-contract --branch master || echo $?
```

Comment:
In fact, `cargo-contract` is [installed](https://github.com/paritytech/scripts/blob/master/dockerfiles/contracts-ci-linux/Dockerfile#L44) in CI image and this line was to install the latest version of it, is it's [done](https://github.com/paritytech/ink/blob/master/.gitlab-ci.yml#L151) in `ink!`'s CI. This was to conduct CI tests against the latest version of `contract`. @ascjones is it necessary here?

Reply:
I get that. But it is done again in `build.sh`. That is the point here.

Reply:
I added it back. I changed build.sh to not reinstall when it is already installed. `build.sh` should also work locally without those re-install annoyances. CI will still get the newest version that way.

Reply:
> But it is done again in `build.sh`

You are right, let's have it in one place only. And it should be reinstalled in case if it's newer.

Reply:
And by "one place" I rather mean `build.sh` to have it in CI and locally.

Reply:
Updated the PR. It is now only in `build.sh`.